### PR TITLE
Create menu item component

### PR DESCRIPTION
**Pull request by helen-v**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/menu/tokens/md-menu-tokens.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    light: colors.blue[70].name,
+    dark: colors.blue[20].name
+  },
+  pressed: {
+    light: colors.blue[80].name,
+    dark: colors.blue[10].name
+  },
+  disabled: {
+    light: colors.gray[40].name,
+    dark: colors.gray[70].name
```
Comment (futurethang) [inline web-components/src/components/menu/tokens/md-menu-tokens.js:23]: available as `--md-disabled-text-color`

**Review by futurethang — CHANGES_REQUESTED** [review]

File: web-components/src/components/menu/tokens/lm-menu-tokens.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  pressed: {
+    "bg": {
+      light: colors.blue[20].name,
+      dark: colors.blue[70].name
+    }
+  },
+  disabled: {
+    light: colors.gray[40].name,
+    dark: colors.gray[70].name
```
[the remark this answers is not in the document]
Comment (helen-v) [reply web-components/src/components/menu/tokens/lm-menu-tokens.js:27]: fixed

File: web-components/src/components/menu/scss/menu-item.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    a {
+      background: var(--menu-pressed-bg, $lm-menu-pressed-bg-light);
+    }
+  }
+
+  &.disabled {
+    pointer-events: none;
+    a {
+      background: transparent;
+      color: var(--menu-disabled, $lm-menu-disabled-light);
```
[the remark this answers is not in the document]
Comment (helen-v) [reply web-components/src/components/menu/scss/menu-item.scss:37]: fixed

File: web-components/src/components/menu/tokens/md-menu-tokens.js (hunk 1 above)
Comment (helen-v) [reply web-components/src/components/menu/tokens/md-menu-tokens.js:23]: fixed

File: web-components/src/components/menu/tokens/lm-menu-tokens.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  pressed: {
+    "bg": {
+      light: colors.blue[20].name,
+      dark: colors.blue[70].name
+    }
+  }
+};
+
+module.exports = menu;
```
Comment (elenagiere) [inline web-components/src/components/menu/tokens/lm-menu-tokens.js:27]: add extra line at the end.

File: web-components/src/components/tabs/Tabs.ts [hunk 5]
```diff
@@ -315,6 +315,7 @@ export class Tabs extends ResizeMixin(RovingTabIndexMixin(LitElement)) {
 
       if (newIndex !== -1) {
         this.updateSelectedTab(newIndex);
+        console.log(newIndex);
```
Comment (elenagiere) [inline web-components/src/components/tabs/Tabs.ts:318]: remove console.log

File: web-components/src/components/menu/scss/menu-item.scss [hunk 6]
```diff
@@ -0,0 +1,40 @@
+.md-menu-item {
+  list-style: none;
+  padding: rem-calc(2);
+
+  a {
+    color: var(--menu-default, $lm-menu-default-light);
+    display: flex;
+    padding: 6px 12px;
```
Comment (elenagiere) [inline web-components/src/components/menu/scss/menu-item.scss:8]: create scss variables for these pixel values in the empty file you created: 
`web-components/src/components/menu/scss/settings.scss`

**Review by elenagiere — COMMENTED** [review]
I Left Comments...

Also, the tab navigation isn't quite perfect with these menu-items. it tabs through the disabled item, and gets stuck on the 3rd item.

File: web-components/src/[sandbox]/examples/menu-item.ts [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+          <span>Cisco Edit</span>
+        </md-menu-item>
+      </div>
+    </md-menu-overlay>
+    <md-menu-item>
+      <md-icon name="alarm_16"></md-icon>
+      <span>Cisco Answer</span>
+    </md-menu-item>
+  </md-menu>
+`;
```
[the remark this answers is not in the document]
Comment (helen-v) [reply web-components/src/[sandbox]/examples/menu-item.ts:45]: Done

File: web-components/src/[sandbox]/sandbox.ts [hunk 8]
```diff
@@ -111,6 +112,13 @@ export class Sandbox extends LitElement {
       <md-theme class="theme-toggle" id="app-theme" ?darkTheme=${this.darkTheme} ?lumos=${this.lumos}>
         ${this.themeToggle()}
         <elix-list-explorer class="explorer">
+          <div class="container" aria-label="md-menu-item">
```
[the remark this answers is not in the document]
Comment (helen-v) [reply web-components/src/[sandbox]/sandbox.ts:115]: Yes of course, when component develop will finish, I move it

File: web-components/src/components/menu/tokens/lm-menu-tokens.js (hunk 4 above)
Comment (helen-v) [reply web-components/src/components/menu/tokens/lm-menu-tokens.js:27]: Done

File: web-components/src/components/menu/scss/menu-item.scss (hunk 6 above)
Comment (helen-v) [reply web-components/src/components/menu/scss/menu-item.scss:8]: Fixed

File: web-components/src/components/tabs/Tabs.ts (hunk 5 above)
Comment (helen-v) [reply web-components/src/components/tabs/Tabs.ts:318]: Fixed

File: web-components/src/components/theme/Theme.ts [hunk 9]
```diff
@@ -269,6 +269,9 @@ export class Theme extends LitElement {
       <div class="theme-wrapper">
         <style>
           ${styles}
+          .theme-wrapper {
+            width: 100%;
+          }
```
Comment (futurethang) [inline web-components/src/components/theme/Theme.ts:274]: Hm, what's the context here, what's this achieving? Can a style tag apply to the parent wrapper like that? Seems like there might be a better practice if a width correction is needed.

**Review by futurethang — CHANGES_REQUESTED** [review]

File: web-components/src/components/tabs/Tabs.ts (hunk 5 above)
Comment (futurethang) [reply web-components/src/components/tabs/Tabs.ts:318]: I still see this here, maybe we're missing newer commits?

**Review by futurethang — DISMISSED** [review]

File: web-components/src/components/theme/Theme.ts (hunk 9 above)
Comment (helen-v) [reply web-components/src/components/theme/Theme.ts:274]: the inner container should be stretched to the parent, but for some reason this does not happen in the storybook, which does not make it possible to stretch the component to the full width of the screen when necessary. So it's just small correction

**Review by helen-v — COMMENTED** [review]
Done

File: web-components/src/components/menu/MenuItem.ts [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+        detail: {
+          id: this.id,
+          key: event.code,
+          srcEvent: event
+        },
+        bubbles: true,
+        composed: true
+      })
+    );
+    console.log(event.code)
```
Comment (yananym) [inline web-components/src/components/menu/MenuItem.ts:115]: please clean the console logs

File: web-components/src/components/menu/Menu.ts [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+import reset from "@/wc_scss/reset.scss";
+import styles from "./scss/module.scss";
+
+type ItemId = Element["id"];
+export const MORE_MENU_ITEM_COPY_ID_PREFIX = "more-menu-item-";
+
+@customElement("md-menu")
+export class Menu extends SlottedMixin(RovingTabIndexMixin(LitElement)) {
+  @property({ type: Boolean }) justified = false;
+  @property({ type: String }) direction = "horizontal"
```
Comment (yananym) [inline web-components/src/components/menu/Menu.ts:25]: in direction="vertical" i have to navigate with left-right arrows instead of up/down

**Review by yananym — CHANGES_REQUESTED** [review]

File: web-components/src/components/menu/MenuItem.ts (hunk 10 above)
Comment (helen-v) [reply web-components/src/components/menu/MenuItem.ts:115]: done

File: web-components/src/components/menu/Menu.ts (hunk 11 above)
Comment (helen-v) [reply web-components/src/components/menu/Menu.ts:25]: Done.

**Review by yananym — APPROVED** [review]

**Pull request merged**